src/main/java/org/opentripplanner/netex/mapping/NetexMapper.java

```diff
@@ -28,15 +28,8 @@
 import org.opentripplanner.netex.mapping.calendar.DatedServiceJourneyMapper;
 import org.opentripplanner.netex.mapping.support.FeedScopedIdFactory;
 import org.opentripplanner.routing.trippattern.Deduplicator;
-import org.rutebanken.netex.model.Authority;
+import org.rutebanken.netex.model.*;
```

Comment:
Make sure you use the Code formatter as specified in https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/docs/Codestyle.md, it doesn't allow wildcard imports